contact.js

```diff
@@ -0,0 +1,11 @@
+document.querySelector('.submit').addEventListener('click', (e) => {
+  const email = document.getElementById('email').value;
+  const errorEmail = document.querySelector('.error');
+
+  const right = (str) => str === str.toLowerCase();
+
+  if (!right(email)) {
+    e.preventDefault();
+    errorEmail.textContent = 'Please check if your email is in lowercase letters!!!';
```

Comment:
[OPTIONAL] You did great job no errors in your code 👍. However 👆, I think your error message is so normal. I mean the color. How about if we change it to RED to display it as more errorish 😉.